src/main/RomanNumeral.java

```diff
@@ -22,7 +22,6 @@ public RomanNumeral() {
 
     public String convert(int number) {
         StringBuilder romanNumber = new StringBuilder();
-
         while (number > 0) {
             for(int arabicNumber : converter.keySet()) {
```

Comment:
Bit inconsistent not having a space between the `for` and the `(` when you have a space between the `while` and the `(`.

Reply:
Line 7: Imagine I asked you to use a `new HashMap()` rather than a `LinkedHashMap`. Ideally the only thing you should have to change is the type on the RHS of the assignment on line 7. 

🔎  Clue: Have a look at what interface `LinkedHashMap` uses & think about the 'L' in Solid.

Reply:
Line 31: Rather than a break in the if statement, can you think of another way to stop the looping? Maybe by extending the conditional statement?